extensions/sdk/src/Rally.ts

```diff
@@ -110,7 +110,7 @@ export class Rally {
    */
   private async storeAttributionCodes() {
     const attribution = await this.getAttributionCodes();
-    if (!(typeof attribution === 'object' && attribution !== null)) {
+    if (!(Object.keys(attribution).length === 0 && attribution.constructor === Object)) {
```

Comment:
IMO typescript takes care of object part and we can just check for non-empty object.

```
if (attribution.length)
```

Reply:
Hm, that seems to just return `undefined`, but this works:

`Object.keys(a).length`